### PR TITLE
Add spaces around operators/tokens

### PR DESCRIPTION
**Pull request by UniQP**

### CODE REVIEW

File: main/src/cgeo/geocaching/settings/AbstractCredentialsAuthorizationActivity.java [hunk 1]
```diff
@@ -212,10 +212,12 @@ protected void enableCheckButtonIfReady() {
      */
     private class EnableStartButtonWatcher implements TextWatcher {
         @Override
-        public void beforeTextChanged(final CharSequence s, final int start, final int count, final int after) {}
+        public void beforeTextChanged(final CharSequence s, final int start, final int count, final int after) {
+        }
 
         @Override
-        public void onTextChanged(final CharSequence s, final int start, final int before, final int count) {}
+        public void onTextChanged(final CharSequence s, final int start, final int before, final int count) {
```
Comment (kumy) [inline main/src/cgeo/geocaching/settings/AbstractCredentialsAuthorizationActivity.java]: Codacy Issue found: [Document empty method body](https://www.codacy.com/app/cgeo/cgeo/file/2635104095/issues/source?bid=3217125&fileBranchId=3319519#l219)

File: main/src/cgeo/geocaching/activity/TokenAuthorizationActivity.java [hunk 2]
```diff
@@ -296,10 +296,12 @@ protected void enableStartButtonIfReady() {
      */
     private class EnableStartButtonWatcher implements TextWatcher {
         @Override
-        public void beforeTextChanged(final CharSequence s, final int start, final int count, final int after) {}
+        public void beforeTextChanged(final CharSequence s, final int start, final int count, final int after) {
+        }
 
         @Override
-        public void onTextChanged(final CharSequence s, final int start, final int before, final int count) {}
+        public void onTextChanged(final CharSequence s, final int start, final int before, final int count) {
```
Comment (kumy) [inline main/src/cgeo/geocaching/activity/TokenAuthorizationActivity.java]: Codacy Issue found: [Document empty method body](https://www.codacy.com/app/cgeo/cgeo/file/2635103748/issues/source?bid=3217125&fileBranchId=3319519#l303)

**Comment by UniQP** [comment]
The codacy issues were not introduced by the PR.

**Comment by Bananeweizen** [comment]
Merging, since there is surely nothing wrong about this PR

**Pull request merged**